contracts/libraries/StorageHelper.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.20;
```

Comment:
Should the bytecode in this file be updated to 0.8.20 as well?

Reply:
update